Library/Formula/libstrophe.rb

```diff
@@ -10,9 +10,9 @@ class Libstrophe < Formula
     cellar :any
     revision 2
     sha256 "b533ba5fbc93054ff666907002045eb057decaa582223610135bf5240f2f579b" => :el_capitan
-    sha256 "2ae7ce796b2f9318c7dd156c9014fcd214ab88d28e9af4b081161fc2166d732e" => :yosemite
-    sha256 "6bd460eafc013874c19fc20734d82a43deba034da23df9bf8e54533b79740686" => :mavericks
-    sha256 "64de9bbabe8ff8fd860d3db9d57eb55a83e5899f469731751f7da613a9155752" => :mountain_lion
+    sha1 "cba04617714a99e25d71ab0efd9d26d046d336c4" => :yosemite
+    sha1 "1f824c9bcb4d8e370e5423d26b75bc926de23afc" => :mavericks
+    sha1 "5ed9df92858ba19d256cfc99c1459649b4840f68" => :mountain_lion
   end
```

Comment:
Please leave the `bottle do` block as is; our bot will update it when pulling. Thanks!